html/web/themes/custom/base_starterkit/templates/system/page-title.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Default theme implementation for page titles.
+ *
+ * Available variables:
+ * - title_attributes: HTML attributes for the page title element.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title: The page title, for use in the actual content.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @ingroup themeable
+ */
+#}
+{{ title_prefix }}
+{% if title %}
+  <h1{{ title_attributes }} class="main-container font-semibold text-4xl md:text-xl lg:text-6xl border-b-4 border-black">{{ title }}</h1>
```

Comment:
Please use similar approach - https://git.drupalcode.org/project/paragraphs/-/blob/8.x-1.x/templates/paragraph.html.twig?ref_type=heads#L42

Define classes as list, and then add to the `title_attributes` object with `.addClass(classes)`